tests/authenticate.bats

```diff
@@ -152,3 +167,31 @@ EOM
   expect_output --substring "authentication required" \
                 "buildah push after buildah logout"
 }
+
+@test "authenticate: --compat-auth-file" {
+  skip_if_no_docker
```

Comment:
Apparently no instance of our CI runs with Docker, so this code is completely untested (and right now, incomplete WRT expected output).

I’ll smoke-test the operation manually, and drop this.

Reply:
Dropped.